src/game/PlayerBots/BattleBotAI.cpp

```diff
@@ -885,6 +889,23 @@ void BattleBotAI::UpdateAI(uint32 const diff)
         if (m_role == BB_ROLE_INVALID)
             AutoAssignRole();
 
+        // add gear to character
+        for (auto itr : sObjectMgr.GetPlayerPremadeGearTemplates())
+        {
+            if (itr.second.requiredClass == me->GetClass())
+            {
+                if (m_hasGear == false)
+                {
+                    std::string str = std::to_string(itr.first);
+                    char* cstr = new char[str.length() + 1];
+                    strcpy(cstr, str.c_str());
+
+                    ChatHandler(me).HandleCharacterPremadeGearCommand(cstr);
```

Comment:
You do not need to invoke chat commands here. Just call the ObjectMgr function to apply template.